src/include/sof/lib/notifier.h

```diff
@@ -25,9 +25,21 @@ enum notify_id {
 	NOTIFIER_ID_DMA_DOMAIN_CHANGE,
 };
 
+enum notify_sync_id {
+	NOTIFIER_ID_SYNC_BUFFER_PRODUCE = 0,	/* struct buffer_cb_transact* */
+	NOTIFIER_ID_SYNC_BUFFER_CONSUME,	/* struct buffer_cb_transact* */
+	NOTIFIER_ID_SYNC_BUFFER_FREE,		/* struct buffer_cb_free* */
+	NOTIFIER_ID_SYNC_DMA_COPY,		/* struct dma_cb_data* */
+	NOTIFIER_ID_SYNC_DMA_IRQ,		/* struct dma_cb_data* */
+	NOTIFIER_ID_SYNC_ALL
+};
```

Comment:
Maybe it'd be better to add `NOTIFIER_ID_SYNC_MAX = NOTIFIER_ID_SYNC_ALL`, because now this enum has 2 purposes - serves as 'broadcast' id and limit for loops

Reply:
done